### DISCUSSION TITLE
Animate.css-Dynamic

### DISCUSSION THREAD

**Discussion by newnexo** [animate-css/animate.css#1514] [General]
Hi there,

I installed Animate.css-Dynamic to run animations when they become visible on the page. It works fine, except for a small detail with entry effects:

If the element to be animated becomes visible while scrolling, it will be visible briefly before disappearing and then reappearing with the animation. Is it possible to make the element invisible until it is animated?

**Comment by KodingKhurram** [comment]
Hello @newnexo,

You can use `aniUtil_dramatic` to make the element invisible until it comes to view. If you meant something else please feel free to ask.

Please read more on this [here](https://animate-dynamic.ga/#dramatic).

**Reply by warengonzaga** [reply]
can you take care of this on your Discussions page? @KodingKhurram

**Reply by KodingKhurram** [reply]
> can you take care of this on your Discussions page? @KodingKhurram

Sure @warengonzaga , When I plan to have a Discussion tab in my repo I will redirect all "Animate.css-Dynamic" discussions.

**Comment by warengonzaga** [comment]
hey there, we don't support Animate.css-Dynamic here kindly proceed to the repo of Animate.css-Dynamic. Thanks!
